packages/lesswrong/lib/collections/jargonTerms/schema.ts

```diff
@@ -0,0 +1,66 @@
+import { schemaDefaultValue } from '../../utils/schemaUtils';
+
+const schema: SchemaType<"JargonTerms"> = {
+  postId: {
+    type: String,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    nullable: false,
+  },
+  term: {
+    type: String,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
+    nullable: false,
+  },
+  humansAndOrAIEdited: {
```

Comment:
This _probably_ wants to be a resolver field that looks at the list of userIds that created revisions for this particular jargonTerm, and just derives who it's been edited by from that (assuming we use a known admin/bot account to create the AI-generated jargonTerms, with a known userId).